swagger/v1/framework_response_reference.yaml

```diff
@@ -0,0 +1,27 @@
+FrameworkResponseReference:
+  type: object
+  required:
+    - data
+  properties:
+    data:
+      oneOf:
+      - type: object
+      - type: 'null'
+      - type: array
```

Comment:
An array of what? May be worth specifying the type of array members.

Reply:
so this reference could be an array of responses or a single response depending on the relationship. But not sure how I could add it as a nested item as its referencing itself?

Reply:
Erm... maybe note something to that effect in the description then?

Reply:
done